File: TCPharmacyFrontend/app/domain/models/request/UserRegisterRequest.ts
export class UserRegisterRequest {
    phoneNumber: string;
    password: string = 'default';
    otp: string[] = ['', '', '', '', '', ''];

    constructor(phoneNumber: string, password: string, otp: string[]) {
        this.phoneNumber = phoneNumber;
        this.password = password;
        this.otp = otp;
    }
}

File: TCPharmacyBackend/src/main/java/vn/edu/iuh/fit/pharmacy/utils/request/UserRegisterRequest.java
package vn.edu.iuh.fit.pharmacy.utils.request;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@NoArgsConstructor
@AllArgsConstructor
@Getter
@Setter
public class UserRegisterRequest {
    public String phoneNumber;
    public String password;
    public int otp;
}
